aten/src/TH/generic/THTensorRandom.cpp

```diff
@@ -51,10 +55,73 @@ void THTensor_(geometric)(THTensor *self, THGenerator *_generator, double p)
   TH_TENSOR_APPLY(real, self, *self_data = (real)THRandom_geometric(_generator, p););
 }
 
+#ifdef TH_BLAS_MKL
+#define BERNOULLI_OMP 10000
+void Bernoulli_generate(const int64_t n, const double p, real *r, THGenerator *_generator)
+{
+  int64_t seed = 17 + THRandom_random(_generator) % 4096;
```

Comment:
Why do we have so few seeds (only 4096 distinct values)?

Reply:
I just follow it from the code of Intel Caffe. Maybe it is useless and harmful. I will remove it.